fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -56,11 +67,21 @@ export const RecipeSearchContainer = ({ rightHandContainer }: Props) => {
     chefSelectors.selectLastFetchOrder(state)
   );
 
+  const [page, setPage] = useState(1);
+
   const debouncedRunSearch = debounce(() => runSearch(), 750);
```

Comment:
Sorry, I missed this on the initial review of #1594 – I think this will need to be wrapped in `useCallback` to preserve the identity of the `debouncedRunSearch` function – at the moment, I think we will be getting a new function every render, which will make the debouncing behaviour a bit unpredictable. We could add this in another PR (which I'm happy to write!)